src/pal/src/exception/signal.cpp

```diff
@@ -697,6 +697,42 @@ PAL_ERROR InjectActivationInternal(CorUnix::CPalThread* pThread)
 
 /*++
 Function :
+    signal_ignore_handler
+
+    Simple signal handler which does nothing
+
+Parameters :
+    POSIX signal handler parameter list ("man sigaction" for details)
+
+(no return value)
+--*/
+static void signal_ignore_handler(int code, siginfo_t *siginfo, void *context)
+{
+}
+
+
+void PAL_IgnoreProfileSignal(void)
+{
+#if !HAVE_MACH_EXCEPTIONS && defined(SIGRTMAX)
```

Comment:
I think that sticking to one signal would be fine. Or can you already see a use case where multiple signals would be useful?